server/config/options.go

```diff
@@ -21,8 +21,9 @@ import (
 )
 
 type ServerConfig struct {
-	Host string
-	Port int16
+	Host      string
+	Port      int16
+	FDBDelete bool `mapstructure:"fdb_delete" yaml:"fdb_delete" json:"fdb_delete"`
```

Comment:
This config param name can be confusing because delete also means deletes done in response to the DELETE query. How about something like `FDBHardDrop`, or something else which makes it clear that this is applicable to Drops only.

Reply:
Sure will update.